scripts/lint.py

```diff
@@ -1120,6 +1120,19 @@ def main() -> None:
         "./scripts/zombie_todos.py",
         "./tests/python/release_checklist/main.py",
         "./web_viewer/re_viewer.js",  # auto-generated by wasm_bindgen
+        # TODO(emilk): remove after parsing all .gitignore files
+        # without this running it locally is very noisy
+        # node_modules
```

Comment:
First: if you write a TODO, add your name to it, not mine. The idea is that is someone has questions about the TODO, they know who to ask.

Second: Try to be as explicit as possible. Remove _what_? What does "this" refer to? What does "it" refer to?

Third: is this meant to read as one sentence or two? It's very difficult to know when you don't use proper capitalization and punctuation.

Reply:
> First: ...

In the moment I thought of it as an extension of the TODO below it, but this should've probably been an issue from the start:
- https://github.com/rerun-io/rerun/issues/6730

> Second: ...
> Third: ...

Sorry about that, the issue is hopefully clear enough